lib/bridge/MatrixHandler.js

```diff
@@ -451,6 +451,26 @@ MatrixHandler.prototype._onAdminMessage = Promise.coroutine(function*(req, event
         yield this.ircBridge.sendMatrixAction(adminRoom, botUser, notice, req);
         return;
     }
+    else if (cmd === "!storepass") {
+        let domain = ircServer.domain;
+        let userId = event.user_id;
+        let notice;
+
+        try {
+            yield this.ircBridge.getStore().storePass(userId, domain, args[0]);
```

Comment:
Are spaces allowed in NickServ passwords? If so, then `args[0]` ain't gonna cut it.